src/libraries/Microsoft.PowerFx.Core/Lexer/TexlLexer.cs

```diff
@@ -82,7 +85,20 @@ public enum Flags
 
         // Pretty Print defaults
         public const string FourSpaces = "    ";
-        public const string LineBreakAndfourSpaces = "\n    ";
+        public const string LineBreakAndfourSpaces = "\n    ";
+
+        // Reserved Keywords

```

Comment:
change comment to: Reserved but currently unused keywords

Reply:
Done.